prov/verbs/src/verbs_domain.c

```diff
@@ -104,6 +108,10 @@ static int fi_ibv_domain_close(fid_t fid)
 
 	ofi_mr_cache_cleanup(&domain->cache);
 
+	if (domain->eq) {
+		fi_ibv_eq_detach_domain(domain);
```

Comment:
check for failure and abort if needed

Reply:
I have added an assert to verify the return value of `fi_ibv_eq_detach_domain()`